docs/GettingStarted.rst

```diff
@@ -438,7 +438,8 @@ Quickly perform key actions through the following applications:
 -  *Getting Started*: See a list of suggested actions you can start.
    See :ref:`Getting Started application <Getting Started>` for more details.
 
--  *Calendar*: Get an overview of calendar events. See :ref:`Calendar application <CalendarApp>` for more details.
+-  *Calendar*: Get an overview of calendar events. See :ref:`Calendar application <CalendarApp>
```

Comment:
seems to be unrelated to the news feature

Reply:
I didn't update this part.

Reply:
Fixed